integrationTests/rabbitmq/testNotifierWithRabbitMQ_test.go

```diff
@@ -12,47 +11,61 @@ import (
 	"github.com/multiversx/mx-chain-core-go/data/outport"
 	"github.com/multiversx/mx-chain-core-go/data/smartContractResult"
 	"github.com/multiversx/mx-chain-core-go/data/transaction"
+	logger "github.com/multiversx/mx-chain-logger-go"
 	"github.com/multiversx/mx-chain-notifier-go/common"
-	"github.com/multiversx/mx-chain-notifier-go/data"
 	"github.com/multiversx/mx-chain-notifier-go/integrationTests"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+var log = logger.GetOrCreate("integrationTests/rabbitmq")
+
 func TestNotifierWithRabbitMQ(t *testing.T) {
+	t.Run("with http observer connnector", func(t *testing.T) {
+		testNotifierWithRabbitMQ(t, common.HTTPConnectorType)
+	})
+
+	t.Run("with ws observer connnector", func(t *testing.T) {
+		testNotifierWithRabbitMQ(t, common.WSObsConnectorType)
+	})
+}
+
+func testNotifierWithRabbitMQ(t *testing.T, observerType string) {
 	cfg := integrationTests.GetDefaultConfigs()
 	cfg.ConnectorApi.CheckDuplicates = true
 	notifier, err := integrationTests.NewTestNotifierWithRabbitMq(cfg)
 	require.Nil(t, err)
 
-	webServer := integrationTests.NewTestWebServer(notifier.Facade, common.MessageQueueAPIType)
+	client, err := integrationTests.CreateObserverConnector(notifier.Facade, observerType, common.MessageQueueAPIType)
+	require.Nil(t, err)
+
+	time.Sleep(time.Second * 2)
```

Comment:
Why this sleep?
Do you use common resources?
If so, we should rather use a mutex instead of sleep

Reply:
Or are you waiting for the observer creator to finish?
If so, we could move the sleep inside `ntegrationTests.CreateObserverConnector` func

Reply:
already added a separate sleep in inside CreateObserverConnector func but forgot to remove from here, removed it now